lib/Catalyst/Plugin/Session.pm

```diff
@@ -624,33 +620,10 @@ sub create_session_id {
     return $sid;
 }
 
-my $counter;
-
-sub session_hash_seed {
-    my $c = shift;
-
-    return join( "", ++$counter, time, rand, $$, {}, overload::StrVal($c), );
-}
+sub session_hash_seed { }
```

Comment:
A subclass could be using this, so it should still return something useful.

Reply:
Providing a reasonable seed seems like a good idea to me:
```
sub session_hash_seed { Crypt::SysRandom::random_bytes(80) }
```

Reply:
32 bytes (256 bits) is plenty.